glib/src/date.rs

```diff
@@ -27,6 +27,7 @@ unsafe impl Sync for Date {}
 
 impl Date {
     #[doc(alias = "g_date_valid_dmy")]
```

Comment:
looks wrong as a doc alias no?

Reply:
Well, the old script was less efficient, I'll remove it. ;)